DataFormats/PatCandidates/src/PackedCandidate.cc

```diff
@@ -369,6 +369,14 @@ void pat::PackedCandidate::setRawCaloFraction(float p) {
     rawCaloFraction_ = 100*p;
 }
 
+void pat::PackedCandidate::setRawHcalFraction(float p) {
+  rawHcalFraction_ = 100*p;
+}
+
+void pat::PackedCandidate::setCaloFraction(float p) {
+  caloFraction_ = 100*p;
+}
```

Comment:
I was looking at some events in 136.876 and see for PFCandidates_particleFlow some cases with  "(ecalEnergy+hcalEnergy)/energy" above 1.28. This will be truncated. 
Do you want to handle this separately to mark an overflow?
Note that there are negative values as well, but they apparently do not go too low.

Reply:
Looking at the distribution over recent simulated files, and discussing with @rmanzoni, I moved to unsigned int, just like for the raw calo fraction. This better covers the range of values encountered. The fraction of events with negative values is small, and they are also not interesting from the point of view of rejecting electrons or muons misidentified as taus, nor for other purposes we could imagine (like potential inclusion in other future jet taggers).